benchmarks/tpcds/schemas/s64da_columnstore/schema.sql

```diff
@@ -0,0 +1,1012 @@
+-- round(double precision, int) does not exist by default
+CREATE FUNCTION round(double precision, integer) RETURNS NUMERIC AS $$
+    SELECT round($1::numeric, $2)
+$$ LANGUAGE sql;
+
+CREATE EXTENSION swarm64da;
```

Comment:
It would be safer to have here `IF NOT EXISTS`

Reply:
sure but none of them have that. so imo that's a separate PR

Reply:
also did it now exactly like the other schemas with a "pre_schema.sql"

Reply:
Since this is now in `pre_schema.sql` it should be removed from here.

Reply:
oops. forgot tpc-ds. now fixed.